packages/odyssey-react/src/components/Table/TableContainer.tsx

```diff
@@ -16,11 +16,12 @@ import { withTheme } from "@okta/odyssey-react-theme";
 import { useOmit } from "../../utils";
 import { Box } from "../Box";
 import styles from "./TableContainer.module.scss";
+import { theme } from "./TableContainer.theme";
 
 export interface TableContainerProps
   extends Omit<
     ComponentPropsWithRef<"figure">,
-    "style" | "className" | "title" | "color"
+    "style" | "className" | "heading" | "color"
```

Comment:
this shouldn't be necessary any longer as I do not believe `heading` is a native DOM attribute so we should be able to remove this!